app/gui/view/component-browser/component-list-panel/grid/src/entry/style.rs

```diff
@@ -5,65 +5,144 @@ use crate::prelude::*;
 use enso_frp as frp;
 use ensogl_core::data::color;
 use ensogl_core::display::shape::StyleWatchFrp;
-use ensogl_core::Animation;
+use ensogl_core::display::style::data::DataMatch;
 use ensogl_derive_theme::FromTheme;
+use ensogl_hardcoded_theme::application::component_browser::component_list_panel as panel_theme;
 use ensogl_hardcoded_theme::application::component_browser::component_list_panel::grid as grid_theme;
 use entry_theme::highlight::selection as selection_theme;
 use grid_theme::entry as entry_theme;
 
 
 
+// =============
+// === Color ===
+// =============
+
+/// Color of the different parts of the entry.
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum Color {
+    /// The final color is "main" color of the component group with the specified alpha value.
+    #[allow(missing_docs)]
+    ComponentGroup { alpha_multiplier: f32 },
+    /// The final color is defined as an arbitrary color in the stylesheet.
+    Arbitrary(color::Lcha),
+}
+
+impl Default for Color {
+    fn default() -> Self {
+        Self::ComponentGroup { alpha_multiplier: 1.0 }
```

Comment:
I would make the default color just black. It's easier to spot the issue then. Here it is possible that by accident I will show that on component group bg and dont see the icon. Do you agree with that? If so, lets just make it `color::Cha::black()`.